lib/paperclip/io_adapters/identity_adapter.rb

```diff
@@ -1,12 +1,12 @@
 module Paperclip
   class IdentityAdapter < AbstractAdapter
-    def new(adapter)
-      adapter
```

Comment:
I've reverted overriding `#new` in favor of the original implementation (overriding initialize to take no arguments and new as an instance method), I agree that it is less confusing. I've poked around a bit but it seems that one would need to refactor a lot to get rid of this adapter. That seems out of scope for this PR.